src/gigs-board/entity/github-repo/board.jsx

```diff
@@ -0,0 +1,234 @@
+/* INCLUDE: "common.jsx" */
+const nearDevGovGigsContractAccountId =
+  props.nearDevGovGigsContractAccountId ||
+  (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+
+const nearDevGovGigsWidgetsAccountId =
+  props.nearDevGovGigsWidgetsAccountId ||
+  (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+
+function widget(widgetName, widgetProps, key) {
+  widgetProps = {
+    ...widgetProps,
+    nearDevGovGigsContractAccountId: props.nearDevGovGigsContractAccountId,
+    nearDevGovGigsWidgetsAccountId: props.nearDevGovGigsWidgetsAccountId,
+    referral: props.referral,
+  };
+
+  return (
+    <Widget
+      src={`${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.${widgetName}`}
+      props={widgetProps}
+      key={key}
+    />
+  );
+}
+
+function href(widgetName, linkProps) {
+  linkProps = { ...linkProps };
+
+  if (props.nearDevGovGigsContractAccountId) {
+    linkProps.nearDevGovGigsContractAccountId =
+      props.nearDevGovGigsContractAccountId;
+  }
+
+  if (props.nearDevGovGigsWidgetsAccountId) {
+    linkProps.nearDevGovGigsWidgetsAccountId =
+      props.nearDevGovGigsWidgetsAccountId;
+  }
+
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+
+  const linkPropsQuery = Object.entries(linkProps)
+    .filter(([_key, nullable]) => (nullable ?? null) !== null)
+    .map(([key, value]) => `${key}=${value}`)
+    .join("&");
+
+  return `/#/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+    linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+/* END_INCLUDE: "common.jsx" */
+
+/* INCLUDE: "shared/lib/gui" */
+const Card = styled.div`
+  &:hover {
+    box-shadow: rgba(3, 102, 214, 0.3) 0px 0px 0px 3px;
+  }
+`;
+
+const CompactContainer = styled.div`
+  width: fit-content !important;
+  max-width: 100%;
+`;
+/* END_INCLUDE: "shared/lib/gui" */
+
+const dataToColumns = (data, columns) =>
+  Object.values(columns).reduce(
+    (registry, column) => ({
+      ...registry,
+
+      [column.id]: [
+        ...(registry[column.id] ?? []),
+
+        ...data.filter((ticket) =>
+          column?.labelSearchTerms.every((searchTerm) =>
+            searchTerm.length > 0
+              ? ticket.labels.some((label) =>
+                  label.name.toLowerCase().includes(searchTerm.toLowerCase())
+                )
+              : true
+          )
+        ),
+      ],
+    }),
+
+    {}
+  );
+
+const withType = (type) => (data) => ({ ...data, type });
+
+const GithubRepoBoard = ({
+  columns,
+  dataTypesIncluded,
+  description,
+  editorTrigger,
+  isEditable,
+  pageURL,
+  repoURL,
+  title,
+}) => {
+  State.init({
+    ticketsByColumn: {},
+  });
+
+  if (repoURL) {
+    const pullRequests = dataTypesIncluded.PullRequest
+      ? (
+          fetch(
+            `https://api.github.com/repos/${repoURL
+              .split("/")
```

Comment:
doing some sanity check and normalization here? for example a repo can starts with `https://`, ends with `/` or `.git`, etc.

Reply:
No, I was in a big hurry, but I think I can include some simple validation into MVP

Reply:
We can also do the normalization in "create community / edit community form", in that case here you can assume the url is in right form